.svgrrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  template: require('./svgr-template'),
+  dimensions: false,
+  replaceAttrValues: {
+    '#FF0000': '{props.fill}',
```

Comment:
note to the reviewer: the icons come with "#FF000" for the fill in the actual svg, but svgr weirdly munges that to lowercase sometimes, and actually converts to rgb/"red" on web. This just covers all cases for all the platforms